### ISSUE TITLE
[YAML] Replace dftool with pdr cli

### ISSUE THREAD

**Issue by trizin** [oceanprotocol/pdr-backend#521]
### Background / motivation

(In yaml-cli2 branch)

The new CLI was initially created as a copy-and-paste of dftool.

There are still some "dftool" artifacts. These should all be "pdr" now.

Details:
```text
(venv) trentmc@tlm-macbook: ~/code/pdr-backend/pdr_backend $ grep -r dftool|grep -v Binary
./cli/test/test_cli_module.py:    with patch("sys.argv", ["dftool", "help"]):
./cli/test/test_cli_module.py:    with patch("sys.argv", ["dftool", "undefined_function"]):
./cli/test/test_cli_module.py:    with patch("sys.argv", ["dftool", "sim", "ppss.yaml"]):
./cli/test/test_cli_arguments.py:    assert "dftool sim: Begin" in out
./cli/cli_arguments.py:    print(f"dftool {command}: Begin")
```

**Comment by trentmc** [comment]
@trizin it should be "pdr" not "pdrcli". 

Also note: I updated the description for thoroughnesss

**Comment by calina-c** [comment]
Closed by #522
